### ISSUE TITLE
One channel, three playlists. One is fine, two ignored by agent.

### ISSUE THREAD

**Issue by stjep** [ZeroQI/YouTube-Agent.bundle#93]
I've downloaded two playlists as a whole from a single channel using the following yt-dlp settings

`yt-dlp -N 10 --restrict-filenames -f 'bv*[ext=mp4]+ba*[ext=m4a]' --download-archive yt-dls-done.txt  https://www.youtube.com/playlist?list=PL56E3EB1DFD4B64A2 -o '%(uploader)s [%(channel_id)s]/%(playlist_title)s [%(playlist_id)s]/%(playlist_index)s - %(title)s [%(id)s].%(ext)s'`

(Note, I used the restrict-filenames flag for only one playlist in case that was the problem but it didn't work).

The files are saved on disk using this naming convention:

`RedLetterMedia [UCrTNhL_yO3tPTdQ5XgmmWjA]/
├── Star Wars - Episode 1 - The Phantom Menace review [PL5919C8DE6F720A2D]
│   ├── 01 - Star Wars - The Phantom Menace Review (Part 1 of 7) [FxKtZmQgxrI].mp4
│   ├── 01 - Star Wars - The Phantom Menace Review (Part 1 of 7) [FxKtZmQgxrI].webp
│   ├── 02 - Star Wars - The Phantom Menace Review (Part 2 of 7) [ZG1AWVLnl48].mp4
│   ├── 02 - Star Wars - The Phantom Menace Review (Part 2 of 7) [ZG1AWVLnl48].webp
│   ├── 03 - Star Wars - The Phantom Menace Review (Part 3 of 7) [IdQwKPVGQsY].mp4
│   ├── 03 - Star Wars - The Phantom Menace Review (Part 3 of 7) [IdQwKPVGQsY].webp
│   ├── 04 - Star Wars - The Phantom Menace Review (Part 4 of 7) [SOlG4T1S2lU].mp4
│   ├── 04 - Star Wars - The Phantom Menace Review (Part 4 of 7) [SOlG4T1S2lU].webp
│   ├── 05 - Star Wars - The Phantom Menace Review (Part 5 of 7) [TBvp1r2UpiQ].mp4
│   ├── 05 - Star Wars - The Phantom Menace Review (Part 5 of 7) [TBvp1r2UpiQ].webp
│   ├── 06 - Star Wars - The Phantom Menace Review (Part 6 of 7) [ORWPCCzSgu0].mp4
│   ├── 06 - Star Wars - The Phantom Menace Review (Part 6 of 7) [ORWPCCzSgu0].webp
│   ├── 07 - Star Wars - The Phantom Menace Review (Part 7 of 7) [fIWKMgJs_Gs].mp4
│   ├── 07 - Star Wars - The Phantom Menace Review (Part 7 of 7) [fIWKMgJs_Gs].webp
│   ├── 08 - Mr. Plinkett's Star Wars Episode 1 - The Phantom Menace 3D review (part 1 of 2) [GHUj90XDYE0].mp4
│   ├── 08 - Mr. Plinkett's Star Wars Episode 1 - The Phantom Menace 3D review (part 1 of 2) [GHUj90XDYE0].webp
│   ├── 09 - Mr. Plinkett's Star Wars Episode 1 - The Phantom Menace 3D review (part 2 of 2) [x3XVv2SAUL4].mp4
│   ├── 09 - Mr. Plinkett's Star Wars Episode 1 - The Phantom Menace 3D review (part 2 of 2) [x3XVv2SAUL4].webp
│   ├── 0 - Star Wars - Episode 1 - The Phantom Menace review [PL5919C8DE6F720A2D].jpg
│   ├── 10 - Phantom Menace Review Interview Outtakes [KbFCM3vK7iY].mp4
│   └── 10 - Phantom Menace Review Interview Outtakes [KbFCM3vK7iY].webp
├── Star_Wars_-_Episode_2_Attack_of_the_Clones_Review [PL56E3EB1DFD4B64A2]
│   ├── 1 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_1_of_3 [KPt1am18lR4].mp4
│   ├── 1 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_1_of_3 [KPt1am18lR4].webp
│   ├── 2 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_2_of_3 [nq3Wlk2w-Zw].mp4
│   ├── 2 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_2_of_3 [nq3Wlk2w-Zw].webp
│   ├── 3 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_3_of_3 [RqYx0cqy0WA].mp4
│   ├── 3 - Mr._Plinkett_s_Attack_of_the_Clones_Review_part_3_of_3 [RqYx0cqy0WA].webp
│   ├── 4 - Mr._Plinkett_s_Star_Wars_Episode_II_Outtakes [rAca2nzw2lo].mp4
│   └── 4 - Mr._Plinkett_s_Star_Wars_Episode_II_Outtakes [rAca2nzw2lo].webp
└── Star Wars - Episode III - Revenge of the Sith Review [PLJ_TJFLc25JTwinvdLo4PP2rTFfTd9Uhz]
    ├── 0 - Star Wars - Episode III - Revenge of the Sith Review [PLJ_TJFLc25JTwinvdLo4PP2rTFfTd9Uhz].jpg
    ├── 1 - Mr. Plinkett's Star Wars Episode III - Revenge of the Sith Review [bYWAHuFbLoc].en.vtt
    ├── 1 - Mr. Plinkett's Star Wars Episode III - Revenge of the Sith Review [bYWAHuFbLoc].f137.mp4.part
    ├── 1 - Mr. Plinkett's Star Wars Episode III - Revenge of the Sith Review [bYWAHuFbLoc].webp
    ├── 2 - Mr. Plinkett's Episode III Review Epilogue - Revenge of Nadine [S_Ho9NYWl_E].jpg
    └── 2 - Mr. Plinkett's Episode III Review Epilogue - Revenge of Nadine [S_Ho9NYWl_E].mp4`

Plex is perfectly happy picking up the first playlist and its content, but ignores the other two. Logs are attached below.

[com.plexapp.system.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8047499/com.plexapp.system.log)

[com.plexapp.agents.youtube.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8047498/com.plexapp.agents.youtube.log)

I also checked and ASS definitely sees the files. Logs below.

[RedLetterMedia [UCrTNhL_yO3tPTdQ5XgmmWjA].filelist.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8047524/RedLetterMedia.UCrTNhL_yO3tPTdQ5XgmmWjA.filelist.log)
[RedLetterMedia [UCrTNhL_yO3tPTdQ5XgmmWjA].scanner.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8047526/RedLetterMedia.UCrTNhL_yO3tPTdQ5XgmmWjA.scanner.log)

**Comment by ZeroQI** [comment]
*.scanner.log show MP4 files for 2 playlists nothing for the third

System log littered with errors but not from my agent... Seem like the Plex installation is dodgy...

Do all file from the 3 playlist show up in Plex? 
If not need the plex.scanner.log files

**Comment by stjep** [comment]
> *.scanner.log show MP4 files for 2 playlists nothing for the third

Indeed, looks like the download actually failed and is a .part instead of a .mp4. Redownloading and scanning library files the mp4 are now picked up, but no metadata is fetched and they don't appear in Plex.

[RedLetterMedia [UCrTNhL_yO3tPTdQ5XgmmWjA].filelist.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8048711/RedLetterMedia.UCrTNhL_yO3tPTdQ5XgmmWjA.filelist.log)
[RedLetterMedia [UCrTNhL_yO3tPTdQ5XgmmWjA].scanner.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8048713/RedLetterMedia.UCrTNhL_yO3tPTdQ5XgmmWjA.scanner.log)

> System log littered with errors but not from my agent... Seem like the Plex installation is dodgy...

I noticed that also, but I've had zero issues with Plex. I have reinstalled it, however.

> Do all file from the 3 playlist show up in Plex?

Only files from one playlist [PL5919C8DE6F720A2D] appear in Plex. The metadata for these is correct. The two other playlists ([PL56E3EB1DFD4B64A2] and [PLJ_TJFLc25JTwinvdLo4PP2rTFfTd9Uhz]) do not appear in Plex at all.

> If not need the plex.scanner.log files

Sure, attached.

[Plex Media Scanner.log](https://github.com/ZeroQI/YouTube-Agent.bundle/files/8048768/Plex.Media.Scanner.log)

**Comment by kodxana** [comment]
Having that same issue.

**Comment by Bit4BitForensics** [comment]
I am having the same issue as well, except in my case it is grouping all of the playlists together that are under a single youtube channel. Did anybody figure this out? No matter what I do it groups all of the playlist folders together even though it identifies them correctly and as separate playlists in the logs.
